lib/rules/no-jquery.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const ember = require('../utils/ember');
+const utils = require('../utils/utils');
+
+//------------------------------------------------------------------------------
+// General rule -  Disallow usage of jQuery
+//------------------------------------------------------------------------------
+
+const message = 'Do not use jQuery';
+const specialisedMessageForThisJquery = 'Do not use `this.$` in components or tests, as it uses jQuery underneath.';
+const ALIASES = ['$', 'jQuery'];
+
+function isThisJquery(node) {
+  return utils.isMemberExpression(node.callee) &&
+    utils.isThisExpression(node.callee.object) &&
+    node.callee.property.name === '$';
+}
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow use of `this.$` on components or tests',
```

Comment:
You should update description of the rule, as now it disallows any jQuery usage. When you'll do this, please revert readme and execute `npm run update` once again.